src/gui/rss/articlelistwidget.cpp

```diff
@@ -98,7 +99,7 @@ void ArticleListWidget::handleArticleRead(RSS::Article *rssArticle)
     if (!item) return;
 
     item->setData(Qt::ForegroundRole, QPalette().color(QPalette::Inactive, QPalette::WindowText));
-    item->setData(Qt::DecorationRole, QIcon(":/icons/sphere.png"));
+    item->setData(Qt::DecorationRole, UIThemeManager::instance()->getIcon("RSS.ReadArticle"));
```

Comment:
I would use something like "RSSView" instead of "RSS".

Reply:
May be `RSSTab`?

Reply:
Tabs is just a way to organise Views/Widgets in container (Main window in this case).

Reply:
They're actually used in RSS Feed List so may be `RSSFeedList`, but `RSSView` seems good to